importScripts/sampleRecords/media.destination.json

```diff
@@ -0,0 +1,168 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "representation_id":"1",
+      "id":"1",
+      "idno":"10388",
+      "ca_objects.type_id":"Photograph",
+      "ca_objects.idno":"P2010.79",
+      "ca_objects.preferred_labels":"TIMBER FELLING IN WA",
+      "ca_object_representations.access":"0",
+      "ca_objects_x_object_representations.is_primary":"1",
+      "ca_object_representations.type_id":"front",
+      "ca_object_representations.mimetype":"image\/tiff",
+      "ca_object_representations.media":"Image is being processed"
```

Comment:
No file here as the source file is bigger than the threshold for processing inline, hence `Image is being processed`